types/types.go

```diff
@@ -180,6 +180,18 @@ const (
 	replicaManagerPrefix  = instanceManagerPrefix + "r-"
 )
 
+const (
+	StateNone       = ""
+	StateGenerating = "generating"
+	StateError      = "error"
+	StateReady      = "ready"
+
+	SupportBundleLabelKey = "longhorn.io/supportbundle"
+
+	AppManager = "support-bundle-manager"
```

Comment:
`AppManager/AppAgent`? This name is too generic.

Reply:
Updated